ENCODE_import_data.py

```diff
@@ -147,11 +148,16 @@ def attachment(path):
     filename = os.path.basename(path)
     mime_type, encoding = mimetypes.guess_type(path)
     major, minor = mime_type.split('/')
-    detected_type = magic.from_file(path, mime=True).decode('ascii')
+    try:
+        detected_type = magic.from_file(path, mime=True).decode('ascii')
+    except AttributeError:
+        detected_type = magic.from_file(path, mime=True)
 
     # XXX This validation logic should move server-side.
     if not (detected_type == mime_type or
+            detected_type == 'application/gzip' and mime_type == 'application/x-tar' or
             detected_type == 'text/plain' and major == 'text'):
+        print(detected_type,mime_type)
```

Comment:
Printed output needs some sort of explanation of what is being printed. If it was for development purposes only, we would like it to be removed from the final PR